clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -93,6 +93,17 @@ public class TopicConfig {
             "deletes the old segments. Default value is -2, it represents `retention.bytes` value to be used. The effective value should always be " +
             "less than or equal to `retention.bytes` value.";
 
+    public static final String REMOTE_LOG_DISABLE_POLICY_CONFIG = "remote.log.disable.policy";
+
+    public static final String REMOTE_LOG_DISABLE_POLICY_DOC = "Determines whether tiered data for a topic should be retained or " +
+            "deleted after tiered storage disablement on a topic. The two valid options are \"retain\" and \"delete\". If retain is " +
```

Comment:
Can you replace \"retain\" and \"delete\" strings with constants that were defined like REMOTE_LOG_DISABLE_POLICY_RETAIN and REMOTE_LOG_DISABLE_POLICY_DELETE? This will avoid any typos in this text later and refer to the actual values.